droid-hal-device.inc

```diff
@@ -421,7 +421,7 @@ fi
 echo Building local tools
```

Comment:
There are 4 more occurrences of `test` still not converted to `[ ]`. 3 above and one way below.